### PR TITLE
Acceptance testing before release 1.9.2

### PR DESCRIPTION
**Pull request by l3ku**
I have made some small overall fixes to the plugin code on this branch, and in my opinion we are ready for a new release. As we have made a lot of changes to the codebase recently, this point is a good opportunity to do some UAT before I merge this next week and tag the new release. Possible points to focus on:
- Is something critically broken in new or old features?
- Do the texts / Finnish translations make any sense?
- Should we add something critical that is missing before this release?

TLDR changelog:
- Drag&drop / open&close postboxes on every page except logs, domains and updates
- Shadows page
- Cruft plugins
- Improved cruftfiles removal (multiple at once, additional file data, UI improvements)
- Cruft themes
- Various bugfixes

If you see a bug somewhere, please report it here or in an issue.

### CODE REVIEW

File: seravo-plugin.php [hunk 1]
```diff
@@ -159,7 +159,7 @@ public static function load_all_modules() {
     /*
      * Instance switcher
      */
-    if ( apply_filters('seravo_show_instance_switcher', true) && getenv('CONTAINER') ) {
+    if ( apply_filters('seravo_show_instance_switcher', true) && getenv('WP_ENV') !== 'development' ) {
```
Comment (ottok) [inline seravo-plugin.php]: JEE! :+1:

**Comment by ottok** [comment]
LGTM

**Comment by mikkoaf** [comment]
Nice!

**Comment by l3ku** [comment]
There is still a major show stopper for this release: https://github.com/Seravo/seravo-plugin/issues/171. I will probably look into the fix this week so we can get the release out.

**Comment by l3ku** [comment]
Made some additional bugfixes to the logs viewer, see commits for changes. There is still 
a PR (https://github.com/Seravo/seravo-plugin/pull/172) waiting to be merged, but this release already has plenty of changes for a minor release so I will merge this now, and https://github.com/Seravo/seravo-plugin/pull/172 can be added to the next release.
